tests/unit-tests/src/test/java/org/apache/activemq/artemis/tests/unit/core/paging/impl/PagingStoreImplTest.java

```diff
@@ -1252,6 +1252,7 @@ public void run() {
          };
          store.applySetting(new AddressSettings().setMaxSizeBytes(1000).setAddressFullMessagePolicy(AddressFullMessagePolicy.BLOCK));
          store.addSize(100);
+         store.flushExecutors();
          store.checkMemory(trackMemoryChecks, null);
          assertEquals(1, calls.get());
```

Comment:
another possible fix would probably be Wait.assertEquals

Reply:
The wait may not work well for example here waiting for zero "0" would not prevent the task being fired later as it's init state is 0.